app/templates/attendee-app.hbs

```diff
@@ -84,7 +84,7 @@
         </div>
       </div>
       <div class="eight wide column">
-        <img class="ui fluid image" src="/images/android-img.jpg" alt="Android Pic">
+        <img class="ui fluid image medium" src="/images/android-img.jpg" alt="Android Pic">
```

Comment:
Please see how the `organiser app` is implemented.

Reply:
@kushthedude , I have reread the Contribution & Best Practices Guide again and have added the fix prefix to the name of this PR. In regards to your comment, I do not understand where I went wrong. There is no external CSS you are using and since you are using inline CSS and Semantic UI, it seemed appropriate to add the correct class to the image.